src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
@andreasohlund use the `NotifyOnFailedMessage` api and then call some other class that can be injected into the container.

but better yet change custom checks to not be enabled by magic assembly scanning